seqr/models.py

```diff
@@ -36,6 +36,16 @@ def _slugify(text):
     return __slugify(text).replace('-', '_')
 
 
+class AnvilUser(models.Model):
```

Comment:
It seems no way to incorporate new attributes into the existing User model unless replacing it with a derivate one. The issue of using a derivate user model requires to update all the files which use the original user model. It could be some consequences in changing the model. Creating a new user model for AnVIL user has a similar issue.  
It seems to be the best practice by using a [one-to-one link](https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model.html) with the existing user model. We could continue to use this one-to-one link after the full migration to AnVIL. I'm not sure about the efficiency yet. But, from a coding point of view, the differences from an incorporated use model are small.

Reply:
Looking into this a bit more, it looks like we could be using RemoteUsers for this, which would solve a bunch of our problems: 
https://docs.djangoproject.com/en/3.1/ref/contrib/auth/#django.contrib.auth.backends.RemoteUserBackend
https://docs.djangoproject.com/en/3.1/howto/auth-remote-user/

Reply:
I'm still investigating the `RemoteUsers`.